hw2/04-background.html

```diff
@@ -19,11 +19,11 @@ <h1>Exercise 04 - Background</h1>
       </p>
       <div>
         <label>Background change interval (seconds): </label>
-        <input />
+        <input type="number" id="time"/> </input>
```

Comment:
This is also a good opportunity to use value=3 and a min= and a max= potentially so they can put in negative numbers or perhaps very large ones